portality/static/js/doaj.fieldrender.edges.js

```diff
@@ -2195,9 +2195,15 @@ $.extend(true, doaj, {
                         this.currentQueryString = qs.queryString || "";
                     }
                 }
-                var frag = "<li class='alert'><p>You searched for <i>'";
+                var frag = "<li class='alert'><p>You searched for ‘<i>";
                 frag += edges.escapeHtml(this.currentQueryString);
-                frag +="'</i> and we found no results.</p><p>Search terms must be in <strong>English</strong>.</p> <p>Try removing some of the filters you have set, modifying the text in the search box, or using less specific search terms.</p></li>";;
+                frag += "</i>’ and we found no results.</p>";
+                frag += "<p>Please check the following:</p><ul>";
+                frag += "<li>Check the spelling and make sure that there are no missing characters.</li>";
+                frag += "<li>Use fewer words in your search to make the search less specific.</li>";
+                frag += "<li>Try removing some of the filters you have set.</li>";
+                frag += "<li>Retry your search in English as much of the index uses English terms.</li>";
+                frag += "</ul></li>";
```

Comment:
Also not a blocker, but for such a large bit of text, we'd be better off using a multi-line string, as it is a bit more efficient in the client

```
frag += "<p>Please check the following:</p><ul>\
    <li>Check the spelling and make sure that there are no missing characters.</li>\
    <li>Use fewer words in your search to make the search less specific.</li>\
";

etc.  That way we can maintain the html indentation too, which makes it easier to read.

Reply:
Done!